ci-operator/config/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master.yaml

```diff
@@ -0,0 +1,67 @@
+base_images:
+  base:
+    name: "4.2"
+    namespace: ocp
+    tag: base
+binary_build_commands: make build --warn-undefined-variables
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.12
+images:
+- dockerfile_path: Dockerfile.rhel
+  from: base
+  inputs:
+    bin:
+      as:
+      - registry.svc.ci.openshift.org/ocp/builder:golang-1.12
+      paths: null
+  to: cluster-policy-controller
+promotion:
+  name: "4.2"
+  namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 9Gi
+    requests:
+      cpu: "3"
+      memory: 6Gi
+tag_specification:
+  name: "4.2"
+  namespace: ocp
+test_binary_build_commands: make build GOFLAGS='-race' --warn-undefined-variables
+tests:
+- as: verify
+  commands: |
+    make verify -k --warn-undefined-variables
+  container:
+    from: bin
+- as: verify-deps
+  commands: |
+    HOME=$( mktemp -d )
+    make verify-deps -k --warn-undefined-variables
+  container:
+    from: src
+- artifact_dir: /tmp/artifacts
+  as: unit
+  commands: |
+    mkdir -p /tmp/artifacts
+    make test-unit JUNITFILE=/tmp/artifacts/unit_report.xml --warn-undefined-variables
+  container:
+    from: test-bin
+- as: e2e-aws
+  commands: TEST_SUITE=openshift/conformance/parallel run-tests
+  openshift_installer:
+    cluster_profile: aws
+- as: e2e-aws-upgrade
+  commands: TEST_SUITE=all run-upgrade-tests
+  openshift_installer:
+    cluster_profile: aws
+    upgrade: true
+- as: e2e-aws-builds
```

Comment:
I don't think we need builds here

Reply:
I've removed, also bumped the config to 4.3

Reply:
I've removed, and bumped to 4.3